packages/nestjs-trpc/lib/factories/middleware.factory.ts

```diff
@@ -30,7 +30,7 @@ export class MiddlewareFactory {
       );
 
       const procedureMiddleware = procedures.flatMap((procedure) => {
-        return procedure.middlewares != null ? procedure.middlewares : [];
+        return procedure.middlewares ?? [];
```

Comment:
[note] this was coming through as `undefined` in one of the tests so I changed it to handle either.

Feel free to resolve